### PR TITLE
"Country" instead of "Location"

### PR DESCRIPTION
**Pull request by pzwsk**

### CODE REVIEW

File: frontend/error_404.html [hunk 1]
```diff
@@ -136,7 +136,7 @@ <h1>Ooops! Page not found</h1>
                         <a href="#" class="btn odfri_btn-custom btn-lg" ng-click="changeview('contribute.html', '1')">Submit a dataset </a>
                     </div>
                     <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6 text-center">
-                        <a href="#" class="btn odfri_btn-custom btn-lg" ng-click="changeview('countries.html', '5')">Explore locations</a>
+                        <a href="#" class="btn odfri_btn-custom btn-lg" ng-click="changeview('countries.html', '5')">Explore countries</a>
```
Comment (oncletom) [inline frontend/error_404.html]: This change should be repeated in the navigation menu of each HTML page located in the folder `frontend`.

**Comment by pzwsk** [comment]
Ok, thanks

File: frontend/error_404.html (hunk 1 above)
Comment (pzwsk) [reply frontend/error_404.html]: Ok, thanks I have overlooked that.

What should I do then? do another commit on issue#142 on GFDRR/open-risk-data-dashboard with required changes?

Comment (oncletom) [reply frontend/error_404.html]: You can continue adding more commits to this branch indeed. They will be added to this pull request 👍

Comment (pzwsk) [reply frontend/error_404.html]: Ok, should be done now, can you have a look and validate, thanks

**Pull request merged**